packages/-ember-data/node-tests/docs/test-coverage.js

```diff
@@ -24,7 +24,7 @@ QUnit.module('Docs coverage', function (hooks) {
   if (!process.env.REUSE_DOCS) {
     buildDocs();
   }
-  const docs = require('../../dist/docs/data.json'); // eslint-disable-line node/no-missing-require
```

Comment:
yep! cc @bmish for fix

Reply:
?

Reply:
@bmish It looks like we were too aggressive in removing the unused require lint rule. When I ran auto fix here it resulted in an unexpected removal.

Reply:
ESLint in [removing](https://eslint.org/docs/user-guide/command-line-interface#--report-unused-disable-directives) the unused disable directives.

My guess is that some people have the `dist` folder locally and some don't causing the disable directive to be unused sometimes but used other times?

Reply:
@bmish right, cause that file is generated at build time only when we generate the docs. I think we need to deactivate that for this line and/or deactivate it generally if it can't be deactivated for one line only.